test/require-config.js

```diff
@@ -6,30 +6,31 @@ require.config({
     baseUrl: '/base',
     waitSeconds: 60,
     paths: {
-        backbone: 'node_modules/backbone/backbone',
+        'backbone': 'node_modules/backbone/backbone',
```

Comment:
Interesting. I don't like quotes around fields either. ESLint has a huge number of configuration options for this rule:

http://eslint.org/docs/rules/quote-props

I think I like the option ``"consistent-as-needed"``, but I'd also be fine with ``"as-needed"`` which is the more lenient version.